src/test/java/seedu/address/logic/commands/EditStudentCommandTest.java

```diff
@@ -57,28 +58,29 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
         Student editedStudent = personInList.withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
                 .withTags(VALID_TAG_HUSBAND).build();
 
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder().withName(VALID_NAME_BOB)
-                .withPhone(VALID_PHONE_BOB).withTags(VALID_TAG_HUSBAND).build();
-        EditCommand editCommand = new EditCommand(indexLastPerson, descriptor);
+        EditStudentCommand.EditStudentDescriptor descriptor = new EditStudentDescriptorBuilder()
```

Comment:
should just use `EditStudentDescriptor `directly instead of referring to `EditStudentCommand`